python/src/mas/cli/install/app.py

```diff
@@ -163,7 +163,7 @@ def formatCatalog(self, name: str) -> str:
     def processCatalogChoice(self) -> list:
         self.catalogDigest = self.chosenCatalog["catalog_digest"]
         self.catalogMongoDbVersion = self.chosenCatalog["mongo_extras_version_default"]
-        if self.architecture != "s390x":
+        if self.architecture != "s390x" or self.architecture != "ppc64le":
```

Comment:
These conditions should be an `and` rather than an `or` shouldn't it?

Reply:
It should be or ,set application for core and manage if it is s390x or arch is power

Reply:
This needs to be an and doesn't it?
i.e. if architecture is ppc64le then self.architecture != "s390x" will be True and as this is an `or` then we will go into the block

Reply:
Sorry , my mistake done